### DISCUSSION TITLE
Can I get transaction from a command?

### DISCUSSION THREAD

**Discussion by sandstone991** [ueberdosis/tiptap#5420] [General Questions] [answered]
Sorry if it's a beginner question

I'm trying to use [appendTransaction](https://prosemirror.net/docs/ref/#state.PluginSpec.appendTransaction), I want to be able to return the transaction itself of the command to be appended to the list of the transactions to be executed.
Can this be done?

**Comment by bdbch** [comment]
Afaik `appendTransaction` is only used in Prosemirror Plugins (which is why it's in the PluginSpec)

**Comment by sandstone991** [comment, marked as answer]
This is more what I meant/wanted https://discuss.prosemirror.net/t/refactor-commands-into-transform-helpers/479/16
